bluesky_widgets/qt/figures.py

```diff
@@ -155,6 +157,10 @@ def _on_title_changed(self, event):
             index = self.indexOf(widget)
             self.setTabText(index, event.value)
 
+    def _on_tab_changed(self, event):
```

Comment:
This method is being used as a call by a Qt Signal (that is, it's a Qt Slot) not a call back for bluesky-live EventEmitters, so I think we should name this parameter `index` or something other than `event`.